src/streaming-services/netflix/NetflixApi.ts

```diff
@@ -257,7 +248,7 @@ class _NetflixApi extends Api {
 			} while (!hasReachedEnd && itemsToLoad > 0);
 			if (historyItems.length > 0) {
 				const historyItemsWithMetadata = await this.getHistoryMetadata(historyItems);
-				items = historyItemsWithMetadata.map(this.parseHistoryItem);
+				items = historyItemsWithMetadata.map((historyItem) => this.parseHistoryItem(historyItem));
```

Comment:
It's because of the changes I made to only use arrow class methods when needed. I thought it was best to keep this method as a normal method (I kept only event listeners as arrow methods), so I had to change this so that `this` wouldn't get unbound.